src/config-schema.js

```diff
@@ -262,6 +262,11 @@ const configSchema = {
         type: 'boolean',
         default: true
       },
+      restorePreviousWindowsOnStartAlways: {
+        description: 'When checked *ALWAYS* restores the last state of all Atom windows.',
```

Comment:
I'd only add `, even if opening a single file with Atom` right before the full stop.